gwu-libraries/scholarspace-hyrax
# T127 Advanced Search Page

Includes:

- Creation & styling of advanced search page
- Implementation of Chosen JS library on search page via chosen-rails gem 
- Addition of sassc-rails gem over deprecated sass-rails
- Restructuring of navigation bar to accommodate "Advanced" button
- Misc edits to homepage CSS needed to work with new navbar HTML

To test:
- Visit `/advanced` through clicking the "Advanced" button in the navbar. 
- Perform a search using the free-form search fields on the left side of the page (`All Fields`, `Creator`, `Title`, `Abstract or Summary` and/or `Keyword`) - ensure results in catalog match search. 
- Perform a search using the `Date Created` search fields on the left side of the page - ensure results in catalog have dates of creation between the set dates. 
- Perform a search using the `additional filters` search fields on the right side of the page to narrow search by facets - ensure results in catalog match search.
